app/src/test/java/org/liberty/android/fantastischmemo/utils/HintUtilTest.java

```diff
@@ -81,11 +84,11 @@ public void tearDown() {
     @Test
     public void generateLetterHintTest() {
         boolean valid = false;
-        String result = hintUtil.generateLetterHint(1, word);
-        String result2 = hintUtil.generateLetterHint(middleHintCounter, word);
-        String result3 = hintUtil.generateLetterHint(pastLastCharacterHintCounter, word);
-        Assert.assertEquals("l _ _ _ _ _ _", result);
-        Assert.assertEquals("la   _ _ _ _", result2);
+        String result = hintUtil.generateLetterHint(1, word, used);
```

Comment:
can you explain why you overwrote an existing test for an existing functionality? If you are going to write a test, you should not overwrite someone else's test. You should create a new one.

Reply:
I overwrote the existing test since it originally worked for the way that the feature was implemented where the letters would be displayed in order each time the user clicks the hint button. My task was to change the letters from appearing in order to appearing randomly and by doing so I also changed the feature in hintUtil as well as the related test code for it. I can create a new test for the letterhint and make it random but then there will be two different tests for the letter hint where only one works with the current code.